### DISCUSSION TITLE
Corrupt files

### DISCUSSION THREAD

**Discussion by TripleFJeff** [gopro/labs#467] [Q&A]
I have a Hero 8 that I setup the same way I have done several other cameras, with Labs Firmware and setup to wake on power, start on speed, and make 12GB files:

!MWAKE=2
!MBOOT="!Lspd"!SAVEspd=!SK32H30!R
M64BT=1
It operates as it should, but the files it makes seem to be corrupt. They won't play with VLC or anything else, and when I try to look at the Metadata I get this. Any idea where to start troubleshooting?

![image](https://github.com/gopro/labs/assets/123906976/9cf2d877-ca09-42e3-a5f9-18e540f95e46)

**Comment by TripleFJeff** [comment]
Also, any way to recover?

**Reply by dnewman-gpsw** [reply]
Start with !RESET to clear the Labs settings.  Is the failure with only one camera as you say "done several other cameras".  Maybe one camera has simply failed.
